iib/workers/tasks/iib_static_types.py

```diff
@@ -37,23 +39,33 @@ class PrebuildInfo(TypedDict, total=False):
     bundle_mapping: Dict[str, List[str]]
     distribution_scope: str
     from_index_resolved: str
-    labels: Optional[Dict[str, str]]
     ocp_version: str
     source_from_index_resolved: str
     source_ocp_version: str
     target_index_resolved: str
     target_ocp_version: str
 
 
-class BundleImage(TypedDict):
-    """Type class referencing data related to bundle image used for mypy checking."""
+class PrebuildInfo(PrebuildInfoBase, total=False):
```

Comment:
Hm, I think we can still use `NotRequired` in Python 3.8 if we use `typing_extensions.TypedDict` instead of `typing.TypedDict`

https://peps.python.org/pep-0655/#usage-in-python-3-11